### DISCUSSION TITLE
Unminified CSS options.

### DISCUSSION THREAD

**Discussion by Esteban-Rocha** [glromeo/esbuild-sass-plugin#65] [Q&A] [answered]
Hello, first of all thanks for the plugin, is a bless.

Now I got a very specific use case where I need to compile unminified CSS, the thing is the plugin adds a comment at the top of the generated CSS with this: `/* sass-plugin-0:/Users/.../hero_main.scss */` and I cannot seem to find an option to change that.

I wonder if anyone knows how this behavior can be changed?

Thanks!

**Comment by glromeo** [comment, marked as answer]
Hi Esteban, I don't think avoiding those comments is possible. They are put there by `esbuild`. Since v2.2 the plugin doesn't use namespaces anymore so those filenames should be relative rather than absolute but if you really need to get rid of them I am afraid the only solution is to post process your output

**Reply by Esteban-Rocha** [reply]
@glromeo Well that's really helpful, at least I know the path to follow there. I'll research and come back in some days/weeks when I got the time to hack into this in case is useful for anyone.

Thanks for your time Gianluca. 👍🏻
